src/sGLOH2.cpp

```diff
@@ -7,16 +7,18 @@
 #include <opencv2/core.hpp>
 #include <opencv2/imgproc.hpp>
 
+// Only use odd sizes for the patch size (circular region of interest)
 // good sizes 31, 41, 51, 61, 71, 81, 91, 101, 111, 121
-constexpr int PATCH_SIZE = 81;
+constexpr int PATCH_SIZE = 121;
 
 sGLOH2::sGLOH2(int m) : m(m) {
     // Initialize any other member variables as needed
 }
 
 void sGLOH2::compute(const cv::Mat& image, std::vector<cv::KeyPoint>& keypoints, cv::Mat& descriptors) {
 
-//    // Convert the image to grayscale if it isn't already
+//    // Did some image preprocessing here it did not have a noticeable effect on the results
```

Comment:
I was wondering about this - I noticed he did this in the code but didn't see anything about it in the paper. If Adding color doesn't do much that seems like an easy optimization :)